src/Commands.pm

```diff
@@ -75,6 +75,7 @@ sub initHandlers {
 	buyer				=> \&cmdBuyer,
 	bs					=> \&cmdBuyShopInfoSelf,
 	c					=> \&cmdChat,
+	cancel				=> \&cmdCancel,
```

Comment:
I agree it is not the best command name but the overall npc buy/sell interaction is pretty messed up, I think having a third command is a good idea, but maybe not 'cancel'.